README.md

```diff
@@ -48,7 +48,13 @@ ExaParser can be installed as below.
 
 ## Usage
 
-1. Open [config](config) and adjust parameters as necessary. The most important ones are listed below.
+1. Exaparser will look in the following locations for the `config` file, and use the first one it finds:
+    - The existing file in the root of this repository, *if installed as editable source*.  This won't work for production installs, and is just for testing scenarios.
+    - Your user's home directory at `~/.exabyte/exaparser/config`
+    - A global system configuration at `/etc/exabyte/exaparser/config`
```

Comment:
Since we have been using `/exabyte` directory in the past - e.g. for celim, webapp, should we just is it instead for consistency?

Reply:
This requires a separate discussion. Tracked for a future discussion. Let us keep the changes for now.